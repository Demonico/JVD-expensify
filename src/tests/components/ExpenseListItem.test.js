import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import expenses from '../fixtures/expenses';
import ExpenseListItem from '../../components/ExpenseListItem';

test('should render ExpenseListItem', () => {
  const wrapper = shallow(<ExpenseListItem {...expenses[1]}/>);
  expect(toJSON(wrapper)).toMatchSnapshot();
});