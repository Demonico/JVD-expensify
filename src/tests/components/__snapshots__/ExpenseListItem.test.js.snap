// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseListItem 1`] = `
<div>
  <h3>
    <Link
      replace={false}
      to="/edit/2"
    >
       
      Mortgage
    </Link>
     
  </h3>
  <p>
    $1,155.23
    -
    January 4th, 1970
  </p>
</div>
`;
